### DISCUSSION TITLE
How to extend users table columns?

### DISCUSSION THREAD

**Discussion by anshulporwal26** [supabase/supabase#2677] [Questions]
I want to add more columns to the `users` table like `is_mentor`. How can I do this? Any help would be appreciated.

**Comment by dshukertjr** [comment]
Hi @anshulporwal26 !

It is not recommended to modify the scheme of `auth.users` table. The best way of doing it is by creating a `users` table in public scheme and add any custom user related data there!

You can read more about best practices around managing user data [here](https://github.com/supabase/supabase/discussions/226#discussioncomment-89148)!

**Reply by anshulporwal26** [reply]
Oh okay. I understand the flow now. Thank you so much.

**Comment by lidrissi** [comment]
You can assign metadata to users on sign up
https://supabase.com/docs/guides/auth/managing-user-data
